<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BranchDetails</class>
 <widget class="QMainWindow" name="BranchDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>700</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Branch Details</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;

color: rgb(85, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblcollegecode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>College Code :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="collegecodecomboBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Select</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblcollegename">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>College Name :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCollegeName">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblcourse">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Course :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCourse">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>471</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>48</number>
    </attribute>
    <row/>
    <column/>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Branch Details</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
